x-pack/plugins/global_search_bar/public/components/search_bar.test.tsx

```diff
@@ -229,12 +242,23 @@ describe('SearchBar', () => {
         fireEvent.click(navSearchOptionToClick);
       });
 
-      expect(trackUiMetric).nthCalledWith(1, 'count', 'search_focus');
-      expect(trackUiMetric).nthCalledWith(2, 'click', [
+      expect(reportUiCounter).nthCalledWith(1, 'global_search_bar', 'count', 'search_focus');
+      expect(reportUiCounter).nthCalledWith(2, 'global_search_bar', 'click', [
         'user_navigated_to_application',
         'user_navigated_to_application_discover',
       ]);
-      expect(trackUiMetric).toHaveBeenCalledTimes(2);
+      expect(reportUiCounter).nthCalledWith(3, 'global_search_bar', 'count', 'search_blur');
+      expect(reportUiCounter).nthCalledWith(4, 'global_search_bar', 'count', 'search_blur'); // FIXME why is blur called twice
+      expect(reportUiCounter).toHaveBeenCalledTimes(4);
+
+      expect(reportEvent).nthCalledWith(1, 'global_search_bar_click_application', {
+        application: 'discover',
+        terms: '',
+      });
+      expect(reportEvent).nthCalledWith(2, 'global_search_bar_blur', {
+        focus_time_ms: expect.any(Number),
+      });
+      expect(reportEvent).toHaveBeenCalledTimes(2);
```

Comment:
Unit test actually tests what gets passed to the telemetry services. Previously it tested just what got passed to our wrapper.